src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -952,7 +1104,7 @@ export default class DataTable extends Vue {
       this.$props.config.reserveExpansionSlot ||
       !!this.data.body.find((row: { nestedContent: any }) => row.nestedContent);
     this.data.body.forEach(row => {
-      this._serializedDataBody.push(serializeRow(row, rowNumber, null));
+      this._serializedDataBody.push(serializeRow(row, rowNumber, 0, null, null, null));
```

Comment:
Probably you can set default values in the arguments just to make these calls cleaner